### DISCUSSION TITLE
Bottom and Top Two Thirds

### DISCUSSION THREAD

**Discussion by pleappleappleap** [rxhanson/Rectangle#1438] [Ideas]
It is possible right now to use First,  Center, and Last Third for both horizontal and vertical.  It is also possible to use First Two Thirds and Last Two Thirds for Horizontal.  Would it be possible to add First Two Thirds and Last Two Thirds for Vertical?  It would help a lot with my two portrait-mode side monitors.

**Comment by rxhanson** [comment]
First Two Thirds and Last Two Thirds should work just like the rest of the thirds commands on vertical. Perhaps could you take a screen recording or screenshots to show your expectation and where Rectangle's implementation is not what you expect?

**Comment by pleappleappleap** [comment]
You are indeed right it works on the vertical screen.  I can't get it to do the vertical two thirds on the horizontal screen though.  I will take screenshots shortly.
